oshdb-api/src/test/java/org/heigit/ohsome/oshdb/api/tests/TestMapAggregateByGeometry.java

```diff
@@ -139,16 +136,16 @@ void testZerofill() throws Exception {
 
   @Test
   void testCombinedWithAggregateByTimestamp() throws Exception {
-    SortedMap<OSHDBCombinedIndex<OSHDBTimestamp, String>, Integer> result =
+    SortedMap<OSHDBCombinedIndex<String, OSHDBTimestamp>, Integer> result =
         createMapReducerOSMEntitySnapshot()
             .timestamps(timestamps1)
-            .aggregateByTimestamp()
             .aggregateByGeometry(getSubRegions())
+            .aggregateByTimestamp(OSMEntitySnapshot::getTimestamp)
```

Comment:
What was the reason to change the order of the aggregation from `timestamp, geometry` to `geometry, timestamp`.

Reply:
fixed in 40dba317 by simplifying the implementation of the "automatic" aggregation methods. As as sideeffect, now all(?) aggregation combinations are allowed and possible regardless of their order, for example: `mapRed.aggregateByGeometry(…).aggregateByTimestamp()` or `mapRed.map(…).aggregateByTimestamp().aggregateByGeometry(…)`